documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -57,31 +55,47 @@ metadata:
       uid: <provider_uid>
   labels:
     createdForProviderType: <provider_type> <2>
+    createdForResourceType: provider <3>
```

Comment:
@ahadas For the secret admitter, to block URL changes.
@RichardHoch and also we need to change the label value to `providers`.

Reply:
> @ahadas For the secret admitter, to block URL changes.

ok, I didn't phrase the question properly - I meant why is it specific to OVA (see `OVA only` below) - it looks like we expect every secret for providers to have the `createdForResourceType` label for the validation-webhook and the `createdForProviderType` for the mutating-webhook, no?

Reply:
@bkhizgiy ^^ Please see @ahadas's  question above.

Reply:
oh yes, didn't saw it was specific to OVA, it should be the same as all other providers. 
so in the case it mention already in another section it can be dropped here.